[2023-12-22T03:26:38.274+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: liverpool_dag.send_email_on_failure manual__2023-12-22T03:23:28.042580+00:00 [queued]>
[2023-12-22T03:26:38.288+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: liverpool_dag.send_email_on_failure manual__2023-12-22T03:23:28.042580+00:00 [queued]>
[2023-12-22T03:26:38.290+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-12-22T03:26:38.310+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): send_email_on_failure> on 2023-12-22 03:23:28.042580+00:00
[2023-12-22T03:26:38.316+0000] {standard_task_runner.py:57} INFO - Started process 323 to run task
[2023-12-22T03:26:38.322+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'liverpool_dag', 'send_email_on_failure', 'manual__2023-12-22T03:23:28.042580+00:00', '--job-id', '476', '--raw', '--subdir', 'DAGS_FOLDER/liverpool.py', '--cfg-path', '/tmp/tmpr1yg3s2z']
[2023-12-22T03:26:38.327+0000] {standard_task_runner.py:85} INFO - Job 476: Subtask send_email_on_failure
[2023-12-22T03:26:38.423+0000] {task_command.py:416} INFO - Running <TaskInstance: liverpool_dag.send_email_on_failure manual__2023-12-22T03:23:28.042580+00:00 [running]> on host d017a9b7b4e6
[2023-12-22T03:26:38.583+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='liverpool_dag' AIRFLOW_CTX_TASK_ID='send_email_on_failure' AIRFLOW_CTX_EXECUTION_DATE='2023-12-22T03:23:28.042580+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-22T03:23:28.042580+00:00'
[2023-12-22T03:26:39.816+0000] {logging_mixin.py:154} INFO - Error al enviar el correo: 'ascii' codec can't encode character '\xf3' in position 28: ordinal not in range(128)
[2023-12-22T03:26:39.820+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-22T03:26:39.869+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=liverpool_dag, task_id=send_email_on_failure, execution_date=20231222T032328, start_date=20231222T032638, end_date=20231222T032639
[2023-12-22T03:26:39.946+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-22T03:26:40.004+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
